aviary/subsystems/mass/flops_based/wing_detailed.py

```diff
@@ -281,7 +281,9 @@ def compute(self, inputs, outputs):
 
                 bte = 8 * np.sum((ea[:-1] + ea[1:]) * dy[:-1] * 0.5)
 
-                inertia_factor[i] = 1 - bte / bt * pod_mass[i] / gross_mass
+                inertia_factor_i = 1 - bte / bt * pod_mass[i] / gross_mass
+                # avoid passing an array into specific index of inertia_factor
+                inertia_factor[i] = inertia_factor_i[0]
```

Comment:
It doesn't look like this should ever become a vector, so if it does, then there is something that should be fixed on one of the variables.